doc/sphinx/conf.py

```diff
@@ -188,13 +188,66 @@ def setup(app):
     # but is it the right place?
     'module',
     'redexpr',
+    'red_expr',
```

Comment:
Would you mind adding the renaming rule so that this is `redexpr` or changing the existing `redexpr` into `red_expr` and removing the `redexpr` above? Let's try to be consistent throughout the manual on the name of grammar tokens, even when they are undefined.

Reply:
I'll make it red_expr to match fullGrammar.